extensions/blocks/podcast-player/style.scss

```diff
@@ -128,7 +128,8 @@ $player-float-background: $light-gray-200;
 	}
 
 	// Apply `secondary` color to the podcast title.
-	.has-secondary { // custom color.
+	.has-secondary {
+		// custom color.
```

Comment:
This applies to the line above. Let's remove it or add it to the existing comment if we need to keep it

Reply:
Updated in https://github.com/Automattic/jetpack/pull/15281/commits/bdbed525ff616f1e87c598a29b9b51aa732badf3